### PR TITLE
Firefox 80 released

### PR DESCRIPTION
**Pull request by afmenez**
Firefox 80 will be released today.

### CODE REVIEW

**Comment by chicoxyzzy** [comment]
Private static methods and accessors test pass in 82 Nightly with javascript.options.experimental.private_fields and javascript.options.experimental.private_methods enabled, though I didn't find related issue on a bug tracker

**Comment by chicoxyzzy** [comment]
image

**Comment by chicoxyzzy** [comment]
also weird that note on private instance method has number 0, but the link is the same as under 9 🤔, but that's another bug

**Comment by chicoxyzzy** [comment]
@afmenez would you be open to update private static methods and accessors test results (to flagged)?

**Comment by afmenez** [comment]
OK, I'll do it later today.

**Comment by afmenez** [comment]
Updated Private Static results for Firefox 60 ESR and 82 Nightly.  Also fixed de "[0] note" by moving it to the first reference.

File: data-esnext.js [hunk 1]
```diff
@@ -569,10 +569,11 @@ exports.tests = [
       */},
       res: {
         ie11: false,
-        firefox75: false,
```
Comment (ljharb) [inline data-esnext.js:572]: why is the firefox75 result being removed? manually verified results should never be removed, even if they're redundant.

File: data-esnext.js [hunk 2]
```diff
@@ -624,11 +629,10 @@ exports.tests = [
...
```
Comment (ljharb) [inline data-esnext.js:627]: same here

File: data-esnext.js [hunk 3]
```diff
@@ -656,7 +660,11 @@ exports.tests = [
...
```
Comment (ljharb) [inline data-esnext.js:659]: and here

**Review by ljharb — CHANGES_REQUESTED** [review]
LGTM pending comments

File: data-esnext.js (hunk 1 above)
Comment (afmenez) [reply data-esnext.js:572]: Restored.

File: data-esnext.js (hunk 2 above)
Comment (afmenez) [reply data-esnext.js:627]: Restored.

File: data-esnext.js (hunk 3 above)
Comment (afmenez) [reply data-esnext.js:659]: Restored.

File: data-esnext.js [hunk 4]
```diff
@@ -569,10 +569,13 @@ exports.tests = [
...
       res: {
         ie11: false,
+        firefox60: false,
+        firefox68: false,
         firefox75: false,
         firefox81: {
           val: 'flagged',
-          note_id: 'ff-private-instance-methods-and-accessors'
+          note_id: 'ff-private-instance-methods-and-accessors',
+          note_html: 'The feature have to be enabled via <code>javascript.options.experimental.private_fields</code> and <code>javascript.options.experimental.private_methods</code> settings under <code>about:config</code>.'
```
Comment (ljharb) [inline data-esnext.js:578]:
```suggestion
          note_html: 'The feature has to be enabled via <code>javascript.options.experimental.private_fields</code> and <code>javascript.options.experimental.private_methods</code> settings under <code>about:config</code>.'
```

**Review by ljharb — APPROVED** [review]

File: data-esnext.js (hunk 4 above)
Comment (afmenez) [reply data-esnext.js:578]: It was already like this, I just moved the note.  But I fixed it anyway.

Comment (ljharb) [reply data-esnext.js:578]: for sure, which is why i stamped anyways :-) <3 for fixing it tho!

**Review by chicoxyzzy — APPROVED** [review]
thank you @afmenez!

**Pull request merged**